run_tests.sh

```diff
@@ -5,8 +5,9 @@ script_location="$(readlink -f "$(dirname "$0")")"
 
 workspace_cargo_manifest="$script_location/Cargo.toml"
 
-cargo test --manifest-path "$workspace_cargo_manifest" --workspace --exclude e2e-tests
+# cargo test --manifest-path "$workspace_cargo_manifest" --workspace --exclude e2e-tests
```

Comment:
not sure whether we want these tests to be run; maybe we comment back in?

Reply:
addressed in 733dcd6